generators/entity/index.js

```diff
@@ -61,7 +61,12 @@ module.exports = EntityGenerator.extend({
             type: Boolean,
             defaults: false
         });
+        this.option('table-name', {
+           desc: 'specify table name that will be used by the entity',
+            type: String
+        });
         this.regenerate = this.options['regenerate'];
+        this.tableName = this.options['table-name'];
```

Comment:
I'm fine with this, but why do we need the entityTableName variable? Will it be used somewhere?